src/models/Account.js

```diff
@@ -57,35 +60,40 @@ export class Account extends EntityModel {
           if (status === 401) {
             console.warn("[401] RevOps API access denied. Update your `publicKey`.")
           } else if (status === 400) {
-            console.warn("[400]RevOps API bad request:", response)
+            console.warn(`[400] RevOps API bad request: ${response}`)
           } else {
-            console.error(`[${status}] RevOps API error:`, response)
+            console.error(`[${status}] RevOps API error: ${JSON.stringify(response)}`)
```

Comment:
@chris-revops This may cause a problem with bad responses: JSON.parse: unexpected character